app/controllers/packages_controller.rb

```diff
@@ -1,19 +1,36 @@
 class PackagesController < ApplicationController
   before_action :set_event, only: [:index, :show]
   before_action :set_package, only: [:show]
+  before_action :set_photographers_present, only: [:index, :show]
 
-  def index
+   def index
     @packages = Package.all
+    @event_time_slots = (1..12).to_a
```

Comment:
Not sure, but does this one need to be an instance variable (with @), or would a local variable be sufficient?

Reply:
It's a local variable now :)